### PR TITLE
clean up of curl examples

### PR DESCRIPTION
**Pull request by hchaps**
adding new curl examples (based on eric's spreadsheet), and moving existing curls from main body to right side panel.

### CODE REVIEW

File: spec/swagger.yaml [hunk 1]
```diff
@@ -603,12 +607,7 @@ paths:
       description: |
         This uploads original source content to Smartling. 
         
-        **Example**
-        ```bash
-          curl -X POST -H "Authorization: Bearer {token}" -F "file=@file.properties;type=text/plain" -F "fileUri=file.properties"  -F "fileType=javaProperties" 'https://api.smartling.com/files-api/v2/projects/{projectId}/file'
-        ```
-
-        The example above will upload your Java properties file directly into the Smartling project identified by the `projectId`. Smartling will ingest this file, parse out the keys and text as strings for translation. At this point, content is ready for translation. 
+        The example shown will upload your Java properties file directly into the Smartling project identified by the `projectId`. Smartling will ingest this file, parse out the keys and text as strings for translation. At this point, content is ready for translation.
```
Comment (jones-smartling) [inline spec/swagger.yaml]: i might say "The curl example provided" so they know its in curl where the specific example is

Comment (hchaps) [reply spec/swagger.yaml]: Fixed.

File: spec/swagger.yaml [hunk 2]
```diff
@@ -1085,6 +1080,11 @@ paths:
         401: *Error401ResponseDefinition
         429: *Error429ResponseDefinition
         500: *Error500ResponseDefinition
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X GET -H "Authorization: Bearer $smartlingToken" 'https://api.smartling.com/files-api/v2/projects/{projectId}/locales/all/file/zip?fileUri=yourfile.json&retrievalType=published'
```
Comment (jones-smartling) [inline spec/swagger.yaml]: should be $smartlingProjectId not {projectId}

File: spec/swagger.yaml [hunk 3]
```diff
@@ -1305,18 +1300,17 @@ paths:
         401: *Error401ResponseDefinition
         429: *Error429ResponseDefinition
         500: *Error500ResponseDefinition
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -H "Authorization: Bearer $smartlingToken" "https://api.smartling.com/files-api/v2/projects/{projectId}/files/list?fileTypes[]=json&uriMask=strings"
```
Comment (jones-smartling) [inline spec/swagger.yaml]: same here

**Comment by jones-smartling** [comment]
@hchaps A number of the curl examples use projectId and not the variable smartlingProjectId. They should be changed and review updated. Let me know if you have questions on this.

File: spec/swagger.yaml (hunk 3 above)
Comment (hchaps) [reply spec/swagger.yaml]: Fixed variables in last commit.

File: spec/swagger.yaml (hunk 2 above)
Comment (jones-smartling) [reply spec/swagger.yaml]: this is still incorrect

File: spec/swagger.yaml [hunk 4]
```diff
@@ -6141,18 +6150,22 @@ paths:
                   key: "job.not.found"
                   message: "Job {translationJobUid} not found for project {projectId}."
         500: *Error500ResponseDefinition
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST \
+            'https://api.smartling.com/jobs-batch-api/v1/projects/$smartlingProjectId/batches' \
+            -H "Authorization: Bearer $smartlingToken"  \
+            -d '{
```
Comment (jones-smartling) [inline spec/swagger.yaml]: there could be escaping issues here, lets try it out and see if its work. @hchaps i can help with this.

File: spec/swagger.yaml [hunk 5]
```diff
@@ -6374,30 +6379,27 @@ paths:
                   message: "Bob {batchUid} not found for project {projectId}."
         401: *Error401ResponseDefinition
         500: *Error500ResponseDefinition
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST \
+            'https://api.smartling.com/jobs-batch-api/v1/projects/$smartlingProjectId/batches/{batchUid}' \
+            -H "Authorization: Bearer {token}"  \
```
Comment (jones-smartling) [inline spec/swagger.yaml]: should be the $smartlingToken

File: spec/swagger.yaml [hunk 6]
```diff
@@ -6527,6 +6529,15 @@ paths:
                   message: "The file is currently being processed and will be available in a few minutes."
         429: *Error429ResponseDefinition
         500: *Error500ResponseDefinition
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST \
+            'https://api.smartling.com/jobs-batch-api/v1/projects/$smartlingProjectId/batches/{batchUid}/file' \
```
Comment (jones-smartling) [inline spec/swagger.yaml]: missing Bearer $smartlingToken

File: spec/swagger.yaml [hunk 7]
```diff
@@ -6527,6 +6529,15 @@ paths:
...
+            -H "Authorization: Bearer {token}" \
```
Comment (jones-smartling) [inline spec/swagger.yaml]: should be $smartlingToken not {token}

File: spec/swagger.yaml (hunk 4 above)
Comment (hchaps) [reply spec/swagger.yaml]: fixed.

File: spec/swagger.yaml (hunk 5 above)
Comment (hchaps) [reply spec/swagger.yaml]: fixed.

File: spec/swagger.yaml (hunk 7 above)
Comment (hchaps) [reply spec/swagger.yaml]: fixed

**Comment by hchaps** [comment]
@dimitrystd : The "Create batch' curl was generating errors. I fixed it w/Kyung. Can you please double check the rest of your curls for Job Batches to make sure they're working ok? thanks.

File: spec/swagger.yaml [hunk 8]
```diff
@@ -6314,23 +6322,14 @@ paths:
...
     post:
-      summary: Execute Batch
+      summary: Execute batch
       description: |-
-        Triggers asynchronous executing Batch.
-        In the Execute flow Batch waits until all files are uploaded and attached or failed on one of these operations.
-        When all these asynchronous processes are completed job `authorize` is triggered if was requested and Batch is turned into the `COMPLETED` status
+        Triggers asynchronous execution of a batch.
+        
+        In the execution flow, the batch waits until all files are uploaded and attached, or fails on one of those operations. When all of these asynchronous processes are completed, job `authorize` is triggered if it was requested, and the batch becomes a status of `COMPLETED`.
```
Comment (dkBrazz) [inline spec/swagger.yaml]: > the batch waits ..., or fails on one of those operations.   

Yeah, the original sentence was quite complicated :)

The batch doesn't fail, it waits until upload and attach processes are completed successfully or failed for any reason.

Comment (hchaps) [reply spec/swagger.yaml]: Fixed.

**Comment by dkBrazz** [comment]
> @dimitrystd : The "Create batch' curl was generating errors. I fixed it w/Kyung. Can you please double check the rest of your curls for Job Batches to make sure they're working ok? thanks.

Fixed one more, others look good

**Comment by hchaps** [comment]
@jones-smartling - Anything else? If not, let me know if I can merge.

File: spec/swagger.yaml (hunk 2 above)
Comment (jones-smartling) [reply spec/swagger.yaml]: still not updated. Should be $smartlingProjectId

File: spec/swagger.yaml (hunk 1 above)
Comment (jones-smartling) [reply spec/swagger.yaml]: i see still one more place it is outdated. Otherwise it is good

**Pull request merged**